tests/utils/inputparams/testparams.go

```diff
@@ -49,6 +49,11 @@ func GetVolumeNameWithTimeStamp(volumeName string) string {
 	return volumeName + "_volume_" + strconv.FormatInt(time.Now().Unix(), 10)
 }
 
+// GetUniqueVolumeName prepares unique volume name with a random generated number
+func GetUniqueVolumeName(volName string) string {
```

Comment:
Borrowed this from #1286  since i too ran into the issue while creating volumes in parallel. will remove this from here if it gets merged before